app/domain/errors.rb

```diff
@@ -306,6 +306,16 @@ module AuthnK8s
         msg:  "Invalid Kubernetes host id: {0}. Must end with <namespace>/<resource_type>/<resource_id>",
         code: "CONJ00048E"
       )
+
+      CaResourceMissing = ::Util::TrackableErrorClass.new(
+        msg:  "Resource {0-resource-id} doesn't have saved resource for cert_id or key_id ",
+        code: "CONJ00063E"
```

Comment:
Those error codes need to be documeneted in a KB/Troubleshooting guide?

Reply:
@orenbm ?

Reply:
not at the moment. there is an effort led by @netacoral but i am not sure that it has started.